osu.Game/Database/ArchiveModelManager.cs

```diff
@@ -893,7 +893,7 @@ private void handleEvent(Action a)
 
         #endregion
 
-        private string getValidFilename(string filename)
+        protected static string getValidFilename(string filename)
```

Comment:
naming is wrong. please check the README for contribution guidelines, which include the checks you should be running before pushing.

Reply:
How do I run checks before pushing? I thought the pending checks are awaited approval to run by contributors.
Sorry for the silly questions, this is my first time contributing to an open-source project so I'm getting exposed to a part of github I haven't usually dealt with on my day-to-day use

Reply:
You can check the documentation here: https://github.com/ppy/osu#code-analysis